docs/wiki/development/windows-provisioning.md

```diff
@@ -4,6 +4,8 @@ The bulk of the development environment provisioning logic is in the `tools\prov
 
 Generating the Visual Studio 2015 solution and building the **osquery** binaries is done via the `tools\make-win64-binaries.bat` batch script. 
 
+<b>Note</b>: Both batch scripts above need to be run from the repo root, i.e their invocation should be exactly as they appear above. Read '<b>Build Process</b>' below for more details on provisioning.
```

Comment:
As this is a markdown file, `.md`, you'll want to use surrounding `**` to give bold emphasis :)